R/latent_model.R

```diff
@@ -83,12 +83,12 @@ epidist_family_model.epidist_latent_model <- function(
   # Really the name and vars are the "model-specific" parts here
   custom_family <- brms::custom_family(
     paste0("latent_", family$family),
-    dpars = family$dpars,
-    links = c(family$link, family$other_links),
-    lb = c(NA, as.numeric(lapply(family$other_bounds, "[[", "lb"))),
-    ub = c(NA, as.numeric(lapply(family$other_bounds, "[[", "ub"))),
+    dpars = c(family$dpars, "pwindow", "swindow"),
```

Comment:
Am I right to think that placing parameters into `dpars` rather than injecting them via Stan shouldn't have any implications for the computational efficiency of the model? At the end of the day it's the same Stan code?

Reply:
yes